spec/core/ics-026-routing-module/UPGRADES.md

```diff
@@ -48,14 +48,12 @@ If an error is returned, then core IBC will revert any changes made by `onChanUp
 
 ```typescript
 function onChanUpgradeInit(
-  order: ChannelOrder,
-  connectionHops: [Identifier],
   portIdentifier: Identifier,
   channelIdentifier: Identifier,
-  counterpartyPortIdentifier: Identifier,
-  counterpartyChannelIdentifier: Identifier,
-  version: string,
-  previousVersion: string) => (version: string, err: Error) {
+  upgradeSequence: uint64,
```

Comment:
ah, spec for ics-004 has this after the proposed upgrade field args, probably should be as you have it here tho